.travis.yml

```diff
@@ -4,5 +4,14 @@ language: R
 sudo: false
 cache: packages
 
+r_binary_packages:
+  - units
+  - sf
```

Comment:
do we really need these? travis should look at the suggests in DESCRIPTION and install sf and units (since sf depends on units) automatically

Reply:
This was an attempt to avoid installing `sf` from source, it is not needed if your other suggestion works.
```suggestion
```